src/services/exprensesService.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+const { getAll } = require('../services/usersServer');
+
+let expenses = [];
```

Comment:
```suggestion
const expenses = [];
```

Reply:
i change this array in deleteExprese and updateExprese